bedrock/firefox/templates/firefox/developer/firstrun.html

```diff
@@ -110,6 +110,41 @@ <h3 id="cssgrid" class="body-title">{{ _('Master CSS Grid') }}</h3>
           <a class="more" rel="external" href="https://mozilladevelopers.github.io/playground/css-grid/">{{ _('Learn more') }}</a>
         </div>
       </div>
+
+      {% if l10n_has_tag('deved_new_features') %}
```

Comment:
We usually add the date or bug to l10n tags `new_features_20180830`. 

It makes it easier for us to figure out how old a tag is and if it can be removed. And it avoids confusion if there are also new features to add next release.